ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-SVC3B.yml

```diff
@@ -1494,9 +1494,9 @@ prefix_lists:
 - name: PL-MLAG-PEER-VRFS
   sequence_numbers:
   - sequence: 10
-    action: permit 10.255.251.6/31
-  - sequence: 20
     action: permit 172.31.11.6/31
+  - sequence: 20
+    action: permit 192.168.44.42/31
```

Comment:
so here really we should be careful regarding the generated config as it should be really the network IP of the subnet - this comes from the fact that the primary IP is the same as the secondary IP